components/gitpod-protocol/src/protocol.ts

```diff
@@ -855,20 +855,6 @@ export type WorkspaceSoftDeletion = "user" | "gc";
 export type WorkspaceType = "regular" | "prebuild";
 
 export namespace Workspace {
-    export function getFullRepositoryName(ws: Workspace): string | undefined {
-        if (CommitContext.is(ws.context)) {
-            return ws.context.repository.owner + "/" + ws.context.repository.name;
-        }
-        return undefined;
-    }
-
-    export function getFullRepositoryUrl(ws: Workspace): string | undefined {
-        if (CommitContext.is(ws.context)) {
-            return `https://${ws.context.repository.host}/${getFullRepositoryName(ws)}`;
```

Comment:
This is just bogus. For BBS there is no symmetry between owner+repo and the repository URL. Instead one need to know if it's a personal or a project repository.